docs/BUILD-CIRCUIT.md

```diff
@@ -62,27 +62,27 @@ More details and options for `zkWasm circuit` are given in [its README](./exampl
 ### Convert halo2 circuit to w3bstream project config
 
 ```bash
-ioctl ws code convert -t "halo2" -i "halo2_wasm_bg.wasm"
+ioctl ws project config -s "postgres://test_user:test_passwd@localhost:5432/test?sslmode=disable" -t "halo2" -i "halo2_wasm_bg.wasm"
```

Comment:
don't need set the datasource

Reply:
The new version of `ioctl` is not released yet, so the `datasource` is still needed. When the new version is released, I will move it.